multiraft/multiraft.go

```diff
@@ -33,6 +33,8 @@ import (
 
 const (
 	noGroup = proto.RaftID(0)
+
+	reqBufferSize = 100
```

Comment:
Is this large enough? You mention in the `RegisterAsync` comments that the handler should not block, but `multiraftServer.RaftMessage` will block if the request channel is full. The alternative to blocking there is to change `multiraftServer` to not use a channel but to have a list of requests and a condition variable.